packages/design-system/theme/src/_tokens.scss

```diff
@@ -1,405 +1,448 @@
-	);
-
-	--color-background-xlight-h: 0;
-	--color-background-xlight-s: 0%;
-	--color-background-xlight-l: 100%;
-	--color-background-xlight: hsl(
-		var(--color-background-xlight-h),
-		var(--color-background-xlight-s),
-		var(--color-background-xlight-l)
-	);
-
-	--color-canvas-dot-h: 204;
-	--color-canvas-dot-s: 15.6%;
-	--color-canvas-dot-l: 87.5%;
-	--color-canvas-dot: hsl(
-		var(--color-canvas-dot-h),
-		var(--color-canvas-dot-s),
-		var(--color-canvas-dot-l)
-	);
-
-	--color-canvas-background-h: 260;
-	--color-canvas-background-s: 100%;
-	--color-canvas-background-l: 99.4%;
-	--color-canvas-background: hsl(
-		var(--color-canvas-background-h),
-		var(--color-canvas-background-s),
-		var(--color-canvas-background-l)
-	);
-
-	--color-json-default: #5045A1;
-	--color-json-null:  var(--color-danger);
-	--color-json-boolean: #1d8ce0;
-	--color-json-number: #1d8ce0;
-	--color-json-string: #5045A1;
-	--color-json-key: var(--color-text-dark);
-	--color-json-brackets: var(--color-text-dark);
-	--color-json-brackets-hover: #1890ff;
-	--color-json-line: #bfcbd9;
-	--color-json-highlight: #E2E5EE;
-
-	--color-sticky-default-background-h: 46;
-	--color-sticky-default-background-s: 100%;
-	--color-sticky-default-background-l: 92%;
-	--color-sticky-default-background: hsl(
-		var(--color-sticky-default-background-h),
-		var(--color-sticky-default-background-s),
-		var(--color-sticky-default-background-l)
-	);
-
-	--color-sticky-default-border-h: 43;
-	--color-sticky-default-border-s: 75%;
-	--color-sticky-default-border-l: 80%;
-	--color-sticky-default-border: hsl(
-		var(--color-sticky-default-border-h),
-		var(--color-sticky-default-border-s),
-		var(--color-sticky-default-border-l)
-	);
-
-	--border-radius-xlarge: 12px;
-	--border-radius-large: 8px;
-	--border-radius-base: 4px;
-	--border-radius-small: 2px;
-	--border-color-base: var(--color-foreground-base);
-	--border-color-light: var(--color-foreground-light);
-
-	--border-style-base: solid;
-	--border-width-base: 1px;
-	--border-base: var(--border-width-base) var(--border-style-base)
-		var(--color-foreground-base);
-
-	--font-size-3xs: 0.625rem;
-	--font-size-2xs: 0.75rem;
-	--font-size-xs: 0.8125rem;
-	--font-size-s: 0.875rem;
-	--font-size-m: 1rem;
-	--font-size-l: 1.125rem;
-	--font-size-xl: 1.25rem;
-	--font-size-2xl: 1.75rem;
-
-	--font-line-height-compact: 1.25;
-	--font-line-height-regular: 1.3;
-	--font-line-height-loose: 1.35;
-	--font-line-height-xloose: 1.5;
-
-	--font-weight-regular: 400;
-	--font-weight-bold: 600;
-	--font-family: 'Open Sans', sans-serif;
-
-	--spacing-5xs: 0.125rem;
-	--spacing-4xs: 0.25rem;
-	--spacing-3xs: 0.375rem;
-	--spacing-2xs: 0.5rem;
-	--spacing-xs: 0.75rem;
-	--spacing-s: 1rem;
-	--spacing-m: 1.25rem;
-	--spacing-l: 1.5rem;
-	--spacing-xl: 2rem;
-	--spacing-2xl: 3rem;
-	--spacing-3xl: 4rem;
-	--spacing-4xl: 8rem;
-	--spacing-5xl: 16rem;
+  --color-primary-h: 7;
+  --color-primary-s: 100%;
+  --color-primary-l: 68%;
+  --color-primary: hsl(
```

Comment:
it looks like you changed a few different colors here in this file... was this intentional? can you double check with Max? Pretty sure the floats make a difference..

Reply:
It resolves to the same hex code, but reverted the changes. Thanks!